docs/learn/learn-nominator.md

```diff
@@ -49,6 +49,65 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen.md#optimizations) for more details.
 
+### Bags-list
+
+Nominating accounts are placed in a semi-sorted list called bags-list. This sorting functionality is extremely 
+important for the [long-term improvements](https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376) 
+of the staking/election system. Bags-list allows up to 50,000 nominators to set their *intention* to nominate, 
```

Comment:
This is only the case with the "current parameters". In principle bags-list can store an infinite number of nodes. We should emphasize: 

'in the current config, we keep 50k in bags list and 22,5k come out as the electing'